wandb/sdk/internal/sender.py

```diff
@@ -723,6 +723,17 @@ def send_request_server_info(self, record: "Record") -> None:
             )
         self._respond_result(result)
 
+    def send_request_job_info(self, record: "Record") -> None:
+        """Respond to a request for a job link."""
+        result = proto_util._result_from_record(record)
+        result.response.job_info_response.sequenceId = (
+            self._job_builder._job_seq_id or ""
+        )
+        result.response.job_info_response.version = (
+            self._job_builder._job_version_alias or ""
+        )
```

Comment:
should the default values or the result of the getter be also part of the job_builder?